lib/features/email/presentation/widgets/html_attachment_previewer.dart

```diff
@@ -0,0 +1,96 @@
+
+import 'package:core/presentation/utils/responsive_utils.dart';
+import 'package:core/presentation/views/html_viewer/html_content_viewer_on_web_widget.dart';
+import 'package:core/presentation/views/responsive/responsive_widget.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:pointer_interceptor/pointer_interceptor.dart';
+import 'package:tmail_ui_user/features/email/presentation/widgets/pdf_viewer/top_bar_attachment_viewer.dart';
+import 'package:tmail_ui_user/main/utils/app_utils.dart';
+
+class HtmlAttachmentPreviewer extends StatelessWidget {
+  const HtmlAttachmentPreviewer({
+    super.key,
+    required this.htmlContent,
+    required this.title,
+    required this.mailToClicked,
+    required this.downloadAttachmentClicked,
+  });
+
+  final String title;
+  final String htmlContent;
+  final void Function(Uri? mailToUri) mailToClicked;
```

Comment:
Please use `typedef ` to define for this function

Reply:
Done